jwql/utils/utils.py

```diff
@@ -75,45 +76,65 @@ def filename_parser(filename):
         r"_(?P<exposure_id>\d+)"\
         r"_(?P<detector>((?!_)[\w])+)"
 
-    # Stage 3 filenames with target ID, e.g. "jw80600-o009_t001_miri_f1130w_i2d.fits"
+    # Stage 2c outlier detection filenames
+    # e.g. "jw94015002002_02108_00001_mirimage_o002_crf.fits"
```

Comment:
I suggest to add all new examples to FILENAME_PARSER_TEST_DATA in `test_utils.py`.

Reply:
Oh, great point! Sorry I forgot this on the first go.